docs/standards/react.md

```diff
@@ -0,0 +1,170 @@
+# React Standards
+
+## Controller-View Pattern
+
+Every feature follows a controller-view-hook pattern:
+
+```
+feature/
```

Comment:
I think we should start from a single file feature/index.tsx and break it down to files when it gets bigger (~200) lines.

It's a unclear where the feature folder will live, something like:

```suggestion
src/domain/feature/
```

```suggestion
src/feature/
```